src/Components/Typeahead/Typeahead.purs

```diff
@@ -51,24 +51,33 @@ defRenderFuzzy = HH.span_ <<< ItemContainer.boldMatches "name"
 defRenderItem :: ∀ r. { name :: String | r } -> HH.PlainHTML
 defRenderItem { name } = HH.text name
 
-type RenderContainer o item eff =
-  Select.State (Fuzzy item) (TA.Effects eff)
+type RenderContainer o item eff
+  = Select.State (Fuzzy item) (TA.Effects eff)
   -> H.ComponentHTML (Select.Query o (Fuzzy item) (TA.Effects eff))
 
 defRenderContainer
   :: ∀ o item eff
    . (Fuzzy item -> HH.PlainHTML)
+  -> Array (H.HTML Void (Select.Query o (Fuzzy item) (TA.Effects eff)))
   -> RenderContainer o item eff
-defRenderContainer renderFuzzy selectState =
+defRenderContainer renderFuzzy addlHTML selectState =
   HH.div
     [ HP.class_ $ HH.ClassName "relative" ]
     if selectState.visibility == Select.Off then []
     else
     [ ItemContainer.itemContainer
         selectState.highlightedIndex
         (renderFuzzy <$> selectState.items)
+        addlHTML
     ]
 
+defRenderContainer_
+  :: ∀ o item eff
+   . (Fuzzy item -> HH.PlainHTML)
+  -> RenderContainer o item eff
+defRenderContainer_ renderFuzzy = defRenderContainer renderFuzzy []
```

Comment:
The possible issue here is that `block_` for us tends to mean the same thing as it does with Halogen HTML, that we're not passing in any `IProp`s, vs skipping one random argument. Perhaps something like `defRenderContainer'` or `defRenderContainerBase` would be better suited? Thoughts @thomashoneyman @foresttoney?

Reply:
I prefer `defRenderContainer'`